client/src/components/SearchBar.jsx

```diff
@@ -0,0 +1,17 @@
+import './SearchBar.css'
+
+function SearchBar() {
+  return (
+    <div className="SearchBar">
```

Comment:
what is your workflow about the namming of className because you used Pascal case here and kebab case line 7...